run.py

```diff
@@ -12,9 +12,17 @@
 import wx
 import wx.lib.wordwrap
 import pywintypes
+import win32com.shell.shell as shell
 from wx.lib.itemspicker import ItemsPicker, IP_SORT_CHOICES, IP_SORT_SELECTED, IP_REMOVE_FROM_CHOICES
 
 vernumber = "v2.5.3"  # Version number
+ASADMIN = 'asadmin'
+
+if sys.argv[-1] != ASADMIN and not bool(ctypes.windll.advpack.IsNTAdmin(0, None)):
+    script = os.path.abspath(sys.argv[0])
+    params = ' '.join([script] + sys.argv[1:] + [ASADMIN])
+    shell.ShellExecuteEx(lpVerb='runas', lpFile=sys.executable, lpParameters=params, nShow=1)
+    sys.exit(0)
```

Comment:
Not a fan of this, looks very confusing.

Plus, the exe manifest already requests for admin privileges, so this isn't really required.